src/Services/Air/AirValidator.js

```diff
@@ -208,6 +208,39 @@ Validator.prototype.flightInfoItem = function (item) {
   }
 };
 
+Validator.prototype.fop = function () {
+  if (!this.params.fop) {
+    throw new AirValidationError.FopMissing();
+  }
+  if (
+    Object.prototype.toString.call(this.params.fop) !== '[object Object]' ||
+    this.params.fop.type !== 'Cash'
+  ) {
+    throw new AirValidationError.FopTypeUnsupported();
+  }
+  return this;
+};
+
+Validator.prototype.ticketNumber = function () {
+  if (!this.params.ticketNumber) {
+    throw new AirValidationError.TicketNumberMissing();
+  }
+  if (!this.params.ticketNumber.match(/^\d{13}/)) {
+    throw new AirValidationError.TicketNumberInvalid();
+  }
+  return this;
+};
+
+Validator.prototype.paramsIsObject = function () {
```

Comment:
Not really, as it checks just input object.
For example, `TerminalService` method `executeCommand` requires just `String`, not an object.